video-player-in-flutter-using-chewie-and-getx/index.md

```diff
@@ -0,0 +1,286 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/video-player-in-flutter-using-chewie-and-getx/
+title: Video player in flutter using Chewie and Getx 
+description: This tutorial will guide you on how to set up and integrate Chewie and Getx in your Flutter application.
+author: kanoi-nikita
```

Comment:
change kanoi-nikita to Naomi-Kanoi

Reply:
> change kanoi-nikita to Naomi-Kanoi

the letters should not be capitalized @kanoinikita 
cc @okeloviolet 

Please be sure to go through the docs again.